circle.yml

```diff
@@ -1,7 +1,16 @@
 machine:
   node:
     version: 0.12.0
+dependencies:
+  pre:
+    - wget https://s3-us-west-2.amazonaws.com/ripple-debs/rippled_0.30-1.deb
+    - sudo dpkg -i rippled_0.30-1.deb
 test:
+  pre:
+    - sudo rippled -a --start --conf "$HOME/$CIRCLE_PROJECT_REPONAME/test/integration/rippled.cfg":
+        background: true
   override:
     - scripts/ci.sh "$CIRCLE_NODE_INDEX" "$CIRCLE_NODE_TOTAL":
         parallel: true
+  post:
+    - sudo killall -r rippled
```

Comment:
> I thought I made the package set /var/lib/rippled/db/ writable by anyone. 
Wasn't aware of that. Strange behavior. 
And there is log file in /var/log dir, but not sure if we need it